portfolio/src/main/webapp/script.js

```diff
@@ -2,20 +2,32 @@
 // This function generates a random quote to display, also making sure that the previously displayed random image is not the same as the current random image.
 function randomizeQuote() {
 
+  const range= 4;    // This constant variable keeps track of the number of quotes I have. 
```

Comment:
Add space and make name more descriptive. Also since range is const I believe it is supposed to be all caps. 

NUM_QUOTES = 4